service/service.go

```diff
@@ -95,19 +93,6 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
-	var appCatalogController *appcatalog.AppCatalog
```

Comment:
Another app-operator in unique would handle appCatalog CRD as well, so we don't need it on thiccc branch.